src_c/controllercompat.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file should not exist. If you exist in the future and are wondering if
+ * this file is cool, it is not.
+ */
+
+#include <SDL.h>
+
+/*
+ * Compat thing for controller (see https://github.com/pygame/pygame/pull/3272)
+ */
+int
+PG_GameControllerRumble(SDL_GameController *gamecontroller,
+                        Uint16 low_frequency_rumble,
+                        Uint16 high_frequency_rumble, Uint32 duration_ms)
+{
+#if SDL_VERSION_ATLEAST(2, 0, 9)
+    return SDL_GameControllerRumble(gamecontroller, low_frequency_rumble,
+                                    high_frequency_rumble, duration_ms);
+#else
+    return -1;
```

Comment:
Just a nitpick (I think I mentioned this in a review on the older PR)

I know we don't use the SDL error string in the cython call of this function, but for consistency sake both branches of this compat function should be setting it. This will be useful if in the future any other code uses this compat function and expects to handle an SDL error message